omnibus/resources/chef-workstation/msi/source.wxs.erb

```diff
@@ -245,10 +245,7 @@
 
     <Feature Id="ExperimentalFeatures" Title="!(loc.FeatureExperimental)" Description="!(loc.FeatureExperimentalDesc)" Level="1000" AllowAdvertise="no" Display="collapse">
       <Feature Id="ExperimentalChefWSApp" Title="!(loc.FeatureExperimentalChefWS)" Description="!(loc.FeatureExperimentalChefWSDesc)">
-        <!-- 2018-10-08  disabling automstart until we're ready to enable it for all platforms
-
-             <ComponentRef Id="ChefWSAppAutostartShortcut" />
-         -->
+        <ComponentRef Id="ChefWSAppAutostartShortcut" />
```

Comment:
this will start the Chef Workstation App? Maybe we need a build to verify this functionality?

Reply:
If everyone is ok with the concept I'll kick off an adhoc

Reply:
ok, let us wait.

Reply:
We are ready to do a build! We want this change for this sprint... 

![tenor-225611003](https://user-images.githubusercontent.com/5712253/73565761-8092ca00-441f-11ea-806e-1b8b018c53c5.gif)